bundles/org.openhab.binding.modbus.e3dc/src/main/resources/OH-INF/thing/power-channel-types.xml

```diff
@@ -75,4 +75,16 @@
 		<description>Charge Level of your attached Battery in Percent</description>
 		<state pattern="%d %unit%" readOnly="true"/>
 	</channel-type>
+	<channel-type id="battery-charged-channel">
+		<item-type>Number:Energy</item-type>
+		<label>Battery Charge</label>
+		<description>Charged energy of battery</description>
+		<state pattern="%.3f %unit%" readOnly="true"/>
```

Comment:
Are you sure 3 positions after the decimal point are reasonable for a home storage battery? Same for below.

Reply:
From technical point I would say yes. Values are delivered in kW/h so 3 decimal points representing Watt level. 
E3DC comes with different battery sizes and smallest is 6.5 kW/h
If 1% is charged channel delivers 0,065 kW/h aka 65 W/h

Reply:
Sounds reasonable!